tensorboard/webapp/plugins/text_v2/data_source/text_v2_types.ts

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+
+export interface BackendRunToTagsMap {
```

Comment:
If you're exposing this as part of the interface of TextDataSource then perhaps remove 'Backend' from the name. That you happen to return the raw Backend response is an implementation detail worth hiding from callers.